extensions/bitwarden/package.json

```diff
@@ -64,7 +71,7 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.25.0",
+    "@raycast/api": "^1.27.1",
```

Comment:
Not  sure if it needed to bump raycast version

Reply:
Go ahead, it will allow you to use additional features of the raycast API.

Reply:
Go ahead, it will allow future devs to use newer features of the raycast API.